x-pack/plugins/observability/server/services/slo/transform_generators/apm_transaction_duration.test.ts

```diff
@@ -25,13 +25,13 @@ describe('APM Transaction Duration Transform Generator', () => {
     });
   });
 
-  it("does not include the query filter when params are 'ALL'", async () => {
+  it("does not include the query filter when params are '*'", async () => {
     const anSLO = createSLO(
       createAPMTransactionDurationIndicator({
-        environment: 'ALL',
-        service: 'ALL',
-        transaction_name: 'ALL',
-        transaction_type: 'ALL',
+        environment: '*',
```

Comment:
Why `'*'` instead of `'ALL'`? I see in PR description you mentioned you changed it but I didn't understand why.

Reply:
I think I've seen this used somewhere but can't recall where. I've checked the rules api but it does not seems to be the case. Maybe @simianhacker you have noticed the usage of `*` somewhere? Or that's definitely not something to use.

I can revert back to `ALL` but it would conflict with services named `all` (low chance but you know), or introduce one value per param, e.g. `SERVICE_ALL` for service, `TRANSACTION_NAME_ALL` for transaction_name, etc...

What do you both think?